### PR TITLE
Update tables SDK t1 to t2

### PR DESCRIPTION
**Pull request by v-yilinhu**
Update file list:

1. ./httpTriggerAutoScale/httpTriggerAutoScale/HttpScaleDown.cs
2. ./httpTriggerAutoScale/httpTriggerAutoScale/ScaleSetManager.cs
3. ./httpTriggerAutoScale/httpTriggerAutoScale/TimerClearStoppedVms.cs
4. ./httpTriggerAutoScale/httpTriggerAutoScale/TimerForScale.cs
5. ./httpTriggerAutoScale/httpTriggerAutoScale/WadMetric.cs
6. ./httpTriggerAutoScale/httpTriggerAutoScale/httpTriggerAutoScale.csproj

@scottaddie,@AlexGhiondea,@mario-guerra and @annatisch for notification.

### CODE REVIEW

**Comment by v-xuto** [comment]
@scottaddie, @AlexGhiondea, @mario-guerra, @annatisch   Could you please review this PR?

File: httpTriggerAutoScale/httpTriggerAutoScale/httpTriggerAutoScale.csproj [hunk 1]
```diff
@@ -7,11 +7,9 @@
     <Compile Remove="Utils.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="0.10.1-preview" />
-    <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="3.0.3" />
+    <PackageReference Include="Azure.Data.Tables" Version="12.0.0-beta.6" />
```
Comment (scottaddie) [inline httpTriggerAutoScale/httpTriggerAutoScale/httpTriggerAutoScale.csproj]: Please update to use the latest stable release. At the time of writing, that's version 12.3.0.

**Review by scottaddie — CHANGES_REQUESTED** [review]

File: httpTriggerAutoScale/httpTriggerAutoScale/httpTriggerAutoScale.csproj (hunk 1 above)
Comment (v-yilinhu) [reply httpTriggerAutoScale/httpTriggerAutoScale/httpTriggerAutoScale.csproj]: @scottaddie, I have updated, please review.

**Review by scottaddie — APPROVED** [review]

**Comment by v-xuto** [comment]
@jongio Please help merge this PR.

**Pull request merged**